hw03_frequency_analysis/top_test.go

```diff
@@ -80,3 +76,52 @@ func TestTop10(t *testing.T) {
 		}
 	})
 }
+
+func TestDifferentLenOfSource(t *testing.T) {
+	tests := []lenTestCases{
+		{
+			source:      "Я короткий текст",
+			description: "Короткий текст меньше 10 слов",
+			expected:    3,
+		},
+		{
+			source:      "Я средний текст ровно на 10 слов без повторов ура",
+			description: "Короткий текст == 10 слов",
+			expected:    10,
+		},
+		{
+			source:      "",
+			description: "Пустая строка",
+			expected:    0,
+		},
+		{
+			source:      "- ,, . ! --",
+			description: "Строка из символов",
+			expected:    0,
+		},
+		{
+			source:      "приветик-пипетик, сам-то как ?",
+			description: "Строка с дефисами",
+			expected:    3,
+		},
+		{
+			source:      "Питон - топчик",
+			description: "Строка с тире",
+			expected:    2,
+		},
+	}
+
+	for _, tc := range tests {
+		tc := tc
+		t.Run(tc.description, func(t *testing.T) {
+			actualLen := len(Top10(tc.source))
+			require.Equal(t, tc.expected, actualLen, Top10(tc.source))
```

Comment:
```suggestion
			require.Equal(t, tc.expected, actualLen)
```
Этот аргумент тут лишний, вы же сравниваете expected с actualLen, только и всего

Reply:
Да это для дебага было, чтобы видеть какой источник был